src/spinrhodirect.f90

```diff
@@ -246,7 +250,16 @@ SUBROUTINE SPINRHOEVECS
 
         IF ( UPNE + DOWNNE .EQ. TOTNE ) BREAKLOOP = 1
 
-        !        print*, UPNE, DOWNNE
+        IF ( VERBOSE >= 2 ) THEN
+          write(*,*)"UPNE, DOWNNE", UPNE, DOWNNE
+          write(*,*)"CHEMPOT", CHEMPOT
+        ENDIF
+
+        IF ( CONT > 2*HDIM ) THEN
+          WRITE(*,*)""
+          CALL ERRORS("SPINRHOEVECS","Maximum iterations reached while filling up &
```

Comment:
I don't understand how this fixes the bug. Why is the condition in https://github.com/lanl/LATTE/pull/27/commits/2ec13e83c93bdba8e633ea4353e233cbe240cb1b#diff-22631eba53f24362fa05665083e9405dR251 not triggered in some cases?